### PR TITLE
Adds a new BEPIS design: Extreme Surgery

### PR DESCRIPTION
**Pull request by Time-Green**
Adds the extra-legal surgeries technode to the BEPIS. Currently contains only one surgery.

Extreme Surgery:
Does something. In a world where all mechanics are relatively predictable, I grabbed the most unpredictable and put them into one surgery. DNA meltdown, polymorph, magic projectiles and random reagents. I'd add the die of fate aswell, but that'd be pushing it. 

## Changelog
:cl:
add: A new rumored BEPIS technology was discovered! You can now perform EXTREME SURGERY.
/:cl:

Like I said, the unpredictable can make stuff engaging. A doctor sees 'Extreme surgery' is available and performs it on a random willing assistant, being clueless of the results. Then the assistants skin turns into a xenomorph and eats the doctor. Or he phases out of reality. Or he just melts.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1053,6 +1053,12 @@
 		return FALSE
 	return ..()
 
+/mob/living/carbon/human/fuck()
```
Comment (Fikou) [inline code/modules/mob/living/carbon/human/human.dm]: erp

File: code/modules/surgery/advanced/extreme.dm [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	steps = list(/datum/surgery_step/stroke,
+				/datum/surgery_step/honk,
+				/datum/surgery_step/mechanic_wrench,
+				/datum/surgery_step/toolbox,
+				/datum/surgery_step/fork)
+
+	possible_locs = list(BODY_ZONE_CHEST)
+
+/datum/surgery_step/stroke
+	name = "stroke belly"
```
Comment (Fikou) [inline code/modules/surgery/advanced/extreme.dm]: erp

File: code/modules/mob/living/carbon/human/human.dm (hunk 1 above)
Comment (Time-Green) [reply code/modules/mob/living/carbon/human/human.dm]: its more like FUCK than fuck~

File: code/modules/surgery/advanced/extreme.dm (hunk 2 above)
Comment (Time-Green) [reply code/modules/surgery/advanced/extreme.dm]: only if you make it

**Comment by HomocidlePancakes** [comment]
Put in the die without some rewards.

**Comment by knacker48** [comment]
So what are the high rolls that makes you wanna sign up for this? Since most of that sounds like death

**Comment by HomocidlePancakes** [comment]
Wiz, syndikit and revolver are the only busted ones to be honest.

**Comment by Time-Green** [comment]
> So what are the high rolls that makes you wanna sign up for this? Since most of that sounds like death

Uhhh. Full heal, skeletons, lesses ash drakes or magic immunity? You honestly raise a fair point, because the odds aren't in your favor. But on the other hand, you can do multiple surgeries and get some rare stuff from what could otherwise be a boring greenshift

**Comment by Krysonism** [comment]
This is just magic at this point, shouldn't science rewards be themed around technology?

**Comment by Tlaltecuhtli** [comment]
shouldnt the steps be randomized?

**Comment by itseasytosee** [comment]
this is no ordinary surgery.....

**Comment by itseasytosee** [comment]
Is stroke just help intent?

**Comment by Time-Green** [comment]
> Is stroke just help intent?

Yes

**Comment by optimumtact** [comment]
this is kinda weird

**Comment by ExcessiveUseOfCobblestone** [comment]
magic

**Comment by MrDoomBringer** [comment]
ITT: Arthur Clarke's third law

**Comment by Shadowflame909** [comment]
I mean Polymorph can really suck

You do this surgery on a willing assistant

They get permanently stuck as a lizard.

GG no RE

**Comment by Time-Green** [comment]
> I mean Polymorph can really suck
> 
> You do this surgery on a willing assistant
> 
> They get permanently stuck as a simple mob lizard.
> 
> GG no RE
> 
> Edit: Yeah theres no chance of doing this surgery again if they get simple mobbed'.
> 
> Tough Luck to all the slimes, cows, and simple mob lizards that come out of this. Since thats the most common mob

It's alot more than just polymorph. Either way I could probably just make the surgery accept any kind of living mob

**Comment by knacker48** [comment]
> It's alot more than just polymorph. Either way I could probably just make the surgery accept any kind of living mob

That could actually be handy as somewhat unreliable way of fixing someone who got staff of changed by a wizard, though I feel most rounds won't last long enough to get to that point unless its dynamic or something

**Comment by XDTM** [comment]
Feels too close to magic to really match surgery's theme...
I'd suggest some new bioware surgeries, or perhaps the ability to generate special traumas

**Comment by HomocidlePancakes** [comment]
Magic is just advanced science.

**Comment by Qustinnus** [comment]
instead make it do soemthing funny like replace someone's organs with something really dumb like giving someone a toaster ass or a blackhole stomach

**Comment by Time-Green** [comment]
This was honestly Soviet Golem level meme PR to me. Maybe I'll do something fun like neck extensions, toaster hearts or spoon arms once I got some more time

**Pull request closed without merging**